### PR TITLE
Validator service

### PR DESCRIPTION
**Pull request by elkuku**
This will move the validator methods in the console commands to a service.

Currently there is a bug caused by the fact that the `delete-user` command uses a different method to validate the username than the `add-user` command.

So you can actually **add** a user but you can't **remove** the user ...

<del>Later in 3.3 this can be nicely injected in the constructor</del> done :wink:

### CODE REVIEW

File: src/AppBundle/Utils/Validator.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AppBundle\Utils;
+
+class Validator
+{
+    public function username($username)
```
Comment (uikolas) [inline src/AppBundle/Utils/Validator.php]: Maybe the name could be: `usernameValidation`?

Comment (Koc) [inline src/AppBundle/Utils/Validator.php]: validateUsername

Comment (elkuku) [inline src/AppBundle/Utils/Validator.php]: Finding the "right" name has always been the hardest part in software design :wink:

Comment (voronkovich) [inline src/AppBundle/Utils/Validator.php]: @elkuku, You right, but a method name in most cases should begin with a verb. :)

**Comment by elkuku** [comment]
This has been updated and corresponding unit tests have been added.

**Comment by javiereguiluz** [comment]
Closing in favor of #615. @elkuku I'm sorry but I couldn't fix the conflicts of your PR while merging, so I needed to create a different PR. Thanks!

**Pull request closed without merging**